### ISSUE TITLE
Orphaned label warning 

### ISSUE THREAD

**Issue by xmrsa** [Altinn/app-frontend-react#2176]
### Description of the bug

(In frontend version 4.4.1)

The title element of a repeating group gets a warning when checking with Wave "A form label is present, but it is not correctly associated with a form control."

When I inspect the title-element in the browser it looks like this:
<label class="fds-label fds-label--md fds-label--medium-weight W8t5I51jCvsv5cDZMmRo">
	<span>Hvilke fjernvarmeverk ble biogassen levert til, og hvor mye biogass ble levert?</span>
	<span> *</span>
</label>
![image](https://github.com/Altinn/app-frontend-react/assets/42466346/640219e3-2be0-4cff-a70e-8ffbe459d87b)

### Steps To Reproduce

This is the element that produced this error for me. Changing the mode from onlyTable to showAll, made the warning go away, but that did of course not give the layout I wanted.
`{
        "id": "S05-4-Group-kraftvarmeverk-998L8K",
        "type": "RepeatingGroup",
        "children": [
          "Header-kraftvarmeverk-gruppe-tittel",
          "S05-4-Input-1JSod1",
          "S05-4-Input-STBWzI"
        ],
        "minCount": 0,
        "maxCount": 50,
        "dataModelBindings": {
          "group": "SkjemaData.kraftvarmeverk"
        },
        "textResourceBindings": {
          "title": "S05_BiogassKraftFjernvarme.Paragraph-BmPDgC.title",
          "add_button_full": "iAJXdcuzF0qf-add_button_full"
        },
        "edit": {
          "openByDefault": true,
          "alwaysShowAddButton": true,
          "editButton": false,
          "saveButton": false,
          "addButton": true,
          "deleteButton": true,          
          "mode":"onlyTable"
        },
        "tableColumns": {
          "S05-4-Input-1JSod1": {
            "editInTable": true,
            "showInExpandedEdit": false,
            "width": "50%",
            "alignText": "left"
          },
          "S05-4-Input-STBWzI": {
            "editInTable": true,
            "showInExpandedEdit": false,
            "width": "30%",
            "alignText": "right"
          }
        },
        "rowsAfter": [
          {
            "readOnly": true,
            "cells": [
              {
                "text": "S05_4_BiogassKraftFjernvarme.Input-rN6Hue.title",
                "width": "50%"
              },
              {
                "component": "S05-4-Input-visning-sum-rN6Hue",
                "width": "30%"
              }
            ]
          }
        ]
      },`

### Additional Information

Described on Slack under universell-utforming 
https://altinn.slack.com/archives/C04CZ3KND6Z/p1718871078986549

**Comment by olemartinorg** [comment]
Her bør vi nok prøve å reprodusere i appen det gjelder, da forskjellige modus på repgruppe kan ha en del å si. Skal ikke se bort ifra at dette kan ha blitt fikset i tiden siden dette ble rapportert.

**Comment by olemartinorg** [comment]
Tested in latest app-frontend, and this is no longer an issue. Closing as completed.
